iiigel.sql

```diff
@@ -165,20 +165,21 @@ CREATE TABLE `modules` (
   `sLanguage` varchar(50) NOT NULL,
   `sIcon` varchar(255) NOT NULL,
   `bIsDeleted` tinyint(1) NOT NULL,
-  `bIsLive` tinyint(1) NOT NULL
+  `bIsLive` tinyint(1) NOT NULL,
+  `sSlug` varchar(255) NOT NULL
```

Comment:
der Slug ist ein Wert, der in den URLs eingefügt wird um die Elemente anzusprechen. Da dürfen nur in URLs erlaubte Zeichen rein logischer Weise ;-). Überlegt mal, wie man die generieren kann...